cardano-testnet/test/cardano-testnet-test/Cardano/Testnet/Test/Cli/Conway/DRepRetirement.hs

```diff
@@ -93,49 +93,6 @@ hprop_drep_retirement = H.integrationRetryWorkspace 2 "drep-retirement" $ \tempA
       drepSKeyFp :: Int -> FilePath
       drepSKeyFp n = tempAbsPath' </> "drep-keys" </> ("drep" <> show n) </> "drep.skey"
 
-  -- Create Drep registration certificates
```

Comment:
Do we still test this work flow?

Reply:
No we don't, but I'll readd such a test shortly afterwards.

Follow-up tracked here: https://github.com/IntersectMBO/cardano-node/issues/5742